superset/assets/src/chart/Chart.jsx

```diff
@@ -58,14 +58,15 @@ const BLANK = {};
 const defaultProps = {
   addFilter: () => BLANK,
   filters: BLANK,
-  setControlValue() {},
+  setControlValue() { },
   triggerRender: false,
   itemClick: () => BLANK
 };
 
 class Chart extends React.PureComponent {
   componentDidMount() {
-    if (this.props.triggerQuery) {
+    const { formData } = this.props;
+    if (this.props.triggerQuery && !formData.show_overlay) {
       this.props.actions.runQuery(
         this.props.formData,
```

Comment:
(question) Why not use `formData` instead of `this.props.formData`
```javascript
this.props.actions.runQuery(formData,false)
```

Reply:
You are right! I have Refactored the code to correct this.